### ISSUE TITLE
Consider updating .gitattributes

### ISSUE THREAD

**Issue by Gramps** [graydwarf/godot-gdscript-linter#1]
Hey there! You may want to consider updating your .gitattributes file so users only download the asset itself; there are some extraneous files your current .gitattributes does not catch. A simpler version would be something like:
```
# Normalize line endings for all files that Git considers text files.
* text=auto eol=lf

# Only include the addons folder when downloading from the Asset Library.
/**        export-ignore
/addons    !export-ignore
/addons/** !export-ignore
```

**Comment by graydwarf** [comment]
Oh! Great suggestion! Fixed. Thank you!
